cdc/capture/http_errors.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package capture
+
+import (
+	"strings"
+
+	"github.com/pingcap/errors"
+	cerror "github.com/pingcap/ticdc/pkg/errors"
+)
+
+// HTTPBadRequestError is some errors that will cause a BadRequestError in http handler
+var HTTPBadRequestError = []*errors.Error{
```

Comment:
It should be possible without exposing out.

Reply:
done